README.md

```diff
@@ -0,0 +1,82 @@
+# sanctuary-scripts
+
+Shell scripts used in multiple Sanctuary projects.
+
+Each script is referenced in the `"bin"` field of __package.json__,
+and symlink names are `sanctuary-`-prefixed to avoid collisions.
+To use `generate-readme`, for example, one would use the path
+__node_modules/.bin/sanctuary-generate-readme__ (or simply
+__sanctuary-generate-readme__ when using npm scripts).
+
+## `check-required-files`
+
+Asserts that the project contains important files such as __LICENSE__ and
+__scripts/prepublish__.
+
+## `generate-readme`
+
+Forwards the given filenames to [`transcribe`][transcribe] then replaces
+all `v:${owner}/${name}` and `v:${owner}/${name}#{identifier}` placeholders
+in the output with appropriate GitHub URLs to produce a Markdown readme.
+
+## `lint-bower-json`
+
+Uses [`remember-bower`][remember-bower] to assert that __bower.json__ exists
+and is in sync with __package.json__.
+
+## `lint-commit-message`
+
+Asserts that the current branch name is short enough to appear in a merge
+commit without the commit summary exceeding 72 characters.
+
+## `lint-es3`
+
+Uses [sanctuary-style][sanctuary-style] to lint ES3 code. Accepts any number
+of arguments to be forwarded to [`eslint`][eslint].
+
+## `lint-es6`
+
+Uses [sanctuary-style][sanctuary-style] to lint ES6 code. Accepts any number
+of arguments to be forwarded to [`eslint`][eslint].
+
+## `lint-json`
+
+Asserts that the specified JSON files exist and are neatly formatted.
```

Comment:
Having put this script through its paces I no longer think involving `git` is a good idea. I'll rework the script to use `cp` and `diff` instead. This will avoid making a mess, and will allow linting of untracked JSON files.